lib/eth.ex

```diff
@@ -13,7 +13,11 @@ defmodule ETH do
       46080211
 
   """
-  Application.put_env(:ethereumex, :url, Application.get_env(:eth, :url, "http://localhost:8545"))
+  Application.put_env(
```

Comment:
Why set the application var here? Cant you put it in config/dev.exs? I am guessing this is a default value?

Reply:
Yea it is there in the original lib https://github.com/izelnakri/eth/blob/main/lib/eth.ex, changed `get_env` to `compile_env` cz elixir 1.14 has warning. Yea looks dev.`exs` and `test`.exs also has the config, here it's just setting the default value.